src/app/index/metadata.rs

```diff
@@ -42,6 +47,11 @@ impl From<id3::Tag> for SongTags {
 			.or_else(|| tag.date_released().map(|d| d.year))
 			.or_else(|| tag.date_recorded().map(|d| d.year));
 		let has_artwork = tag.pictures().count() > 0;
+		let lyricist = tag.get_text("TEXT");
+		let composer = tag.get_text("TCOM");
+		let genre = tag.genre().map(|s| s.to_string());
+		let category = tag.get_custom_field("Category");
```

Comment:
I don't think `Category` is a standard or widespread field. Unless I am incorrect here, I would prefer not to include it. The example value in the swagger file is `Filmi` which I would expect to find under Genre (similar to game or movie soundtracks).

Sources:
https://wiki.hydrogenaud.io/index.php?title=Tag_Mapping
https://picard-docs.musicbrainz.org/en/appendices/tag_mapping.html

Reply:
I dropped category. 

(I guess I will have to maintain a patch for category. Maybe game and movie sound reasonable to you as genre but to some of us they are not. Even the genre as defined by id3 standards is not sufficient for genre itself. case in point: id3 has "classical" but India has at least two major classical forms and each having multiple sub-genre. For long I have been stuffing "filmi" in different tag fields. Rust/polaris allowing me(a firmware/embedded guy) to write a music server with ease) so maybe category can be a new 'custom" field.)

Reply:
>  Even the genre as defined by id3 standards is not sufficient for genre itself. case in point: id3 has "classical" but India has at least two major classical forms and each having multiple sub-genre.
The ID3 genre list ([this disaster](https://en.wikipedia.org/wiki/ID3#Genre_list_in_ID3v1[12])) is a leftover from ID3v1 which isn't used much these days. I wholeheartedly agree that would be inadequate, even for popular European/American music. In current versions, the `Genre` field is a text frame (`TCON`) that supports multiple values (see https://id3.org/id3v2.4.0-frames)
You can tag a song with both `Karnāṭaka` and `Tanam` in the Genre field, and even more values if necessary. At least that's what I do for genre and subgenres in my collection.